alts/src/main/java/io/grpc/alts/GoogleDefaultChannelCredentials.java

```diff
@@ -61,16 +66,26 @@ public static ChannelCredentials create() {
     return CompositeChannelCredentials.create(nettyCredentials, callCredentials);
   }
 
+  @SuppressWarnings("unchecked")
   private static InternalProtocolNegotiator.ClientFactory createClientFactory() {
     SslContext sslContext;
     try {
       sslContext = GrpcSslContexts.forClient().build();
     } catch (SSLException e) {
       throw new RuntimeException(e);
     }
+    Attributes.Key<String> clusterNameAttrKey = null;
+    try {
+      Class<?> klass = Class.forName("io.grpc.xds.XdsAttributes");
+      clusterNameAttrKey =
+          (Attributes.Key<String>) klass.getField("ATTR_CLUSTER_NAME").get(null);
+    } catch (ClassNotFoundException | NoSuchFieldException | IllegalAccessException e) {
+      logger.log(Level.FINE, "Unable to load xDS endpoint cluster name key, this may be expected");
```

Comment:
Include `e` in the log statement.

I thought we were avoiding multicatch, but I see that ALTS already has a multicatch usage...

Reply:
Yeah, multicatch is a problem for some old Android. But ALTS is not used in Android, so...

Reply:
It was a blanket avoidance, to avoid having to track everywhere we can and can't use it. For example, we don't use it in tests even though those mostly don't run on Android.

Reply:
Agreed, fixed now.